File: aspnet-core/src/doan.ProjectManagement.Application.Contracts/Projects/Dto/ProjectDto.cs
using doan.ProjectManagement.Enum;
using System;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.Projects.Dto
{
    public class ProjectDto : EntityDto<Guid>
    {
        public string ProjectName { get; set; }
        public ProjectType Type { get; set; }
        public Status Status { get; set; }
        public string Description { get; set; }
        public Level Level { get; set; }
        public int? NumberOfTeamRegister { get; set; }
        public int? LimitSubscriptions { get; set; }
    }
}

File: applications/admin-web/src/app/shared/services/project/project/models/project-dto.ts
import { EntityDto } from '@abp/ng.core';
import { Status } from '../../../system-configuration/system-setting/models';
import { Level } from './level';
import { ProjectType } from './project-type';

export class ProjectDto extends EntityDto<string>{

  projectName: string;
  type: ProjectType;
  status: Status;
  description: string;
  level: Level;
  numberOfTeamRegister: number;
  limitSubscriptions: number;

  constructor(initial: Partial<ProjectDto> = {}) {
    super(initial);
  }
}
